src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseChecklistService.cs

```diff
@@ -169,7 +170,7 @@ private async Task<bool> ReleaseSectionHasNonEmptyHtmlBlock(Guid releaseId, Cont
                     cb.ContentSection!.Release.ReleaseId == releaseId &&
                     cb.ContentSection.Type == sectionType)
                 .OfType<HtmlBlock>()
-                .AnyAsync(htmlBlock => !string.IsNullOrEmpty(htmlBlock.Body));
```

Comment:
Same comment as above, I think this will throw an exception producing the release checklist. I suggest we don't make this change and have a chat to consider whether `StringExtensions.IsNullOrEmpty` is worth any value, given that it can't be translated to SQL out-of-the-box.

Reply:
Yeh fair point, it's unfortunate that it's a bit of a LINQ footgun.

Personally, I'm not against having this, as a similar thing applies for most extension methods we might want to use in a LINQ context. Kinda feels a bit unfair to single this out specifically.

If people feel particularly strongly about it, happy for us to go with whatever though.